vnext/docs/ViewManagers.md

```diff
@@ -240,4 +256,290 @@ AppRegistry.registerComponent('ViewManagerSample', () => ViewManagerSample);
 
 ## Sample ViewManager (C++)
 
-*TODO*
+For this sample, assume we already have the `CustomUserControl` defined in the C# example.
+
+#### 1. Authoring your View Manager 
+
+Here is a sample view manager written in C++ called `CustomUserControlViewManager`.
+
+*CustomUserControlViewManager.h*
```

Comment:
It's hard to maintain the relationship between document and code. I suggest you to generate this md file automatically.

Reply:
The documentation is to be a more descriptive example, while the sample in sample app showcases more examples/features.